_docs/_latest/use/index.md

```diff
@@ -19,3 +19,8 @@ Use the CLI to interface with Forseti and all of its modules.
 **[Get Help using Forseti]({% link _docs/latest/use/get-help.md %})**
 
 Find the best way to get help using Forseti.
+
+**[Using Instrumentation]
```

Comment:
The dev setup needs to be documented, to enable tracing: such as the new API that needs to be enabled, optional packages that needs to be installed, flag that needs to be run.  Using tracing as part of the development flow would be awesome.

https://forsetisecurity.org/docs/latest/develop/dev/setup.html

Reply:
Done